docs/pages/docs/handbook/deploying-with-docker.mdx

```diff
@@ -4,6 +4,11 @@ import Callout from '../../../components/Callout'
 
 Building a [Docker](https://www.docker.com/) image is a common way to deploy all sorts of applications. However, doing so from a monorepo has several challenges.
 
+<Callout>
+  This command is not yet implemented for `npm`.
+</Callout>
```

Comment:
It probably makes more sense to mvoe this closer to where `turbo prune` is called. I'm not sure if lack of pruning means you can't do _anything_ with docker?

Reply:
@mehulkar You can't do anything in the article which is mentioned.

Also in the turborepo migration it suggests beginners use npm, so I guess it means a lot of people will be using npm.